### PR TITLE
Re-enable USING_VARIABLE_LIVE_RANGE

### PR DESCRIPTION
**Pull request by EgorBo**

### CODE REVIEW

File: src/coreclr/jit/emit.h [hunk 1]
```diff
@@ -190,6 +190,8 @@ class emitLocation
 
     UNATIVE_OFFSET GetFuncletPrologOffset(emitter* emit) const;
 
+    bool emitLocation::IsPreviousInsNum(emitter* emit) const;
```
Comment (jkotas) [inline src/coreclr/jit/emit.h:193]:
```suggestion
    bool IsPreviousInsNum(emitter* emit) const;
```

**Comment by BruceForstall** [comment]
@dotnet/jit-contrib

**Comment by EgorBo** [comment]
PTAL @dotnet/jit-contrib (Unfortunately I couldn't manage to merge it before my vacation 😞)
I verified it on arm-crossgen vs arm-crossgen-x64 on Linux-arm64, Linux-arm32 and OSX-arm64

**Review by AndyAyersMS — APPROVED** [review]
LGTM.

**Pull request merged**